tests/s3/test_object_tagging.py

```diff
@@ -1606,6 +1542,7 @@ def test_mpupload_with_tagvalue256chars_42776(self):
         self.log.info("Verify Multipart Upload with tag having "
                       "tag values up to 256 Unicode characters in length")
 
+    @pytest.mark.parallel
```

Comment:
Let's not introduce this marker, unless you have verified parallel execution of the tests